community/values/src/test/java/org/neo4j/values/storable/PointTest.java

```diff
@@ -110,6 +111,19 @@ public void shouldBeAbleToParsePointWithUnquotedCrs()
                 PointValue.parse( "{latitude: 40.7128, longitude: -74.0060, height: 567.8, crs:wgs-84-3D}" ) ); // - explicitly WGS84-3D, without quotes
     }
 
+    @Test
+    public void shouldNotBeAbleToParsePointWithConflictingCrsInformation()
+    {
+        try
+        {
+            PointValue.parse( "{latitude: 40.7128, longitude: -74.0060, height: 567.8, crs:wgs-84-3D}", "wgs-84" );
+        }
+        catch ( IllegalArgumentException e )
+        {
+            assertEquals( "Conflicting crs attributes for point found. Header crs states 'wgs-84' while column specifies 'wgs-84-3D'", e.getMessage() );
```

Comment:
I personally think that it would be useful to have the default in the header and then be able to override it per value. We _could_ even keep the same field format as a Point value to allow for other default values, but I am not sure if we should.